### ISSUE TITLE
nmap crashes on Windows 10 using slow comprehensive scan

### ISSUE THREAD

**Issue by Rewt0r** [nmap/nmap#249]
```
Faulting application name: nmap.exe, version: 7.0.0.0, time stamp: 0x5648b3e8
Faulting module name: nmap.exe, version: 7.0.0.0, time stamp: 0x5648b3e8
Exception code: 0xc0000005
Fault offset: 0x000834f6
Faulting process ID: 0x814
Faulting application start time: 0x01d12d045b4d58d7
Faulting application path: C:\Program Files (x86)\Nmap\nmap.exe
Faulting module path: C:\Program Files (x86)\Nmap\nmap.exe
Report ID: a5c9a894-30a3-48bc-9efc-35591b4a4590
Faulting package full name: 
Faulting package-relative application ID: 
```

Happy to run a debug build if it helps.

**Comment by dmiller-nmap** [comment]
I believe we addressed this in r35551 (ee048d5) and r35552 (7300cde):

<blockquote>
o [Nsock] Avoid a crash on Windows and other systems which use nbase's
  vasprintf implementation. In the case of unknown OpenSSL errors,
  ERR_reason_error_string would return NULL, which could not be printed with
  the "%s" format string. Fixed by using ERR_error_string instead, which will
  not return NULL, and by causing NULL pointers to be printed as "(null)", as
  glibc and MSVC both do. Reported by Dan Baxter. [Gisle Vanem, Daniel Miller]
</blockquote>

I will keep looking at it to see if there's an NSE-only workaround to restore functionality without rebuilding a new installer for Windows. In the meantime, you may try **disabling the mrinfo script** by adding "and not mrinfo" to the `--script` option: `--script "default or (discovery and safe) and not mrinfo"`

**Comment by gvanem** [comment]
I think this issue can be closed. AFAICS, the r35552 fixes this bug.
@Rewt0r can you confirm?

**Comment by Rewt0r** [comment]
Is that part of 7.01 or do I need to use another release from somewhere else?

**Comment by dmiller-nmap** [comment]
@Rewt0r The fix mentioned here was released with Nmap 7.10 in March 2016. The latest version is Nmap 7.12. Does this version work for you, or are you still having problems? We would like to close this bug if it has been fixed.

**Comment by dmiller-nmap** [comment]
Closing, assuming this is resolved.
